packages/cmd/src/help/WebHelpGenerator.ts

```diff
@@ -264,18 +263,14 @@ export class WebHelpGenerator {
     private buildChildrenSummaryTables(helpGen: DefaultHelpGenerator, fullCommandName: string): string {
         const hrefPrefix = fullCommandName + "_";
         return helpGen.buildChildrenSummaryTables().split(/\r?\n/g)
-            .map((line: string, index: number) => {
+            .map((line: string) => {
                 // Wrap group/command names inside links
                 const match = line.match(/^\s{0,4}([a-z0-9-]+(?:\s\|\s[a-z0-9-]+)*)\s+\S/i);
                 if (match) {
                     const href = `${hrefPrefix}${match[1].split(" ")[0]}.html`;
-                    return `\n* <a href="${href}">${match[1]}</a> -` + line.slice(match[0].length - 2);
-                } else if (/^\s*#{4}\s*COMMANDS\s*$/i.test(line)) {
-                    return `${(index > 0) ? "\n" : ""}<h4>Commands</h4>\n`;
-                } else if (/^\s*#{4}\s*GROUPS\s*$/i.test(line)) {
-                    return `${(index > 0) ? "\n" : ""}<h4>Groups</h4>\n`;
```

Comment:
This "hack" is no longer needed, because I fixed the casing of "Commands" and "Groups" in DefaultHelpGenerator.